scripts/gen_eval_files.lua

```diff
@@ -261,7 +261,7 @@ local function get_api_keysets()
   local ret = {} --- @type table<string, vim.EvalFn>
 
   --- @type {[1]: string, [2]: {[1]: string, [2]: string}[] }[]
-  local keysets = metadata.keysets
+  local keysets = metadata.keysets or {}
```

Comment:
Why was this changed? The field should exist.

If this was to avoid building Nvim first, this isn't ideal as it'll wipe the `_meta/api_keysets.lua` file.